### DISCUSSION TITLE
新版本预计什么时候有发布版

### DISCUSSION THREAD

**Discussion by Ptilopsis01** [Blinue/Magpie#756] [General]
如题，似乎新版本有较大的变化，不知道什么时候可以用上，或者什么时候会有预览版发布？急急急.jpg

**Comment by Blinue** [comment]
很快会发布 v0.10.4，没有新功能。v1.0 要重写渲染系统，还需要很久。等不及的话可以在 [Actions](https://github.com/Blinue/Magpie/actions) 里下载开发版。

**Reply by Ptilopsis01** [reply]
ok了解了
